go/client/cmd_configure_nix.go

```diff
@@ -0,0 +1,383 @@
+// Copyright 2019 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build !darwin,!windows
+
+package client
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/install"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+)
+
+const backtick = "`"
+
+type CmdConfigureAutostart struct {
```

Comment:
The filename and struct names should probably have `Sys` in them too, to avoid confusion.